src/json_ui.cc

```diff
@@ -12,6 +12,7 @@
 
 #include <utility>
 
+#include <stdio.h>
```

Comment:
Please use `<cstdio>` if possible, it's the idiomatic way of including C headers into C++ code.

Reply:
Fixed.